proto/palomachain/paloma/evm/query.proto

```diff
@@ -41,6 +43,13 @@ service Query {
     option (google.api.http).get =
         "/palomachain/paloma/evm/query_get_smart_contract_deployments";
   }
+
+  // Get a list of user smart contracts
+  rpc GetUserSmartContracts(QueryUserSmartContractsRequest)
```

Comment:
I think we should try to keep Request/Response objects in line with terminology.

```suggestion
  rpc QueryUserSmartContracts(QueryUserSmartContractsRequest)
```